### DISCUSSION TITLE
Yaml metadata when converting docx to jats

### DISCUSSION THREAD

**Discussion by denismaier** [jgm/pandoc#10289] [Q&A] [answered]
I want to convert a docx to jats xml while pulling in metadata from a yaml file. Is there a way to do that? I don't think I can simply do this

```
pandoc meta.yml content.docx -t jats
```

**Comment by massifrg** [comment, marked as answer]
Try this (see the docs [here](https://pandoc.org/MANUAL.html#option--metadata-file)):

```sh
pandoc --metadata-file=meta.yml content.docx -t jats
```

**Reply by denismaier** [reply]
Cool. I'm already using this when converting docx to md, but I didn't know it also works for other formats. Thanks!

**Comment by estedeahora** [comment]
Perhaps some of the solutions I used for the [`~!guri_`](https://github.com/estedeahora/guri/) project might be useful to you. I think I suspect from your interventions that you are looking for a tool similar to this one. There is a readme in english, although the documentation is not advanced outside spanish (but fortunately there is google translate!).

**Reply by denismaier** [reply]
Thanks for this… I'll have a look. The project looks really interesting.
